lib/runtime/test_helpers.go

```diff
@@ -80,30 +80,24 @@ func GetAbsolutePath(targetDir string) string {
 }
 
 // GetRuntimeBlob checks if the test wasm @testRuntimeFilePath exists and if not, it fetches it from @testRuntimeURL
-func GetRuntimeBlob(testRuntimeFilePath, testRuntimeURL string) (n int64, err error) {
+func GetRuntimeBlob(testRuntimeFilePath, testRuntimeURL string) error {
 	if utils.PathExists(testRuntimeFilePath) {
-		return 0, nil
+		return nil
 	}
 
-	out, err := os.Create(testRuntimeFilePath)
+	/* #nosec */
+	resp, err := http.DefaultClient.Get(testRuntimeURL)
 	if err != nil {
-		return 0, err
+		return err
 	}
-	defer func() {
-		_ = out.Close()
-	}()
 
-	/* #nosec */
-	resp, err := http.Get(testRuntimeURL)
+	respBody, err := io.ReadAll(resp.Body)
 	if err != nil {
-		return 0, err
+		return err
 	}
-	defer func() {
-		_ = resp.Body.Close()
-	}()
+	defer resp.Body.Close() //nolint
```

Comment:
Specify the linter you want disabled. My guess is

```suggestion
	defer resp.Body.Close() //nolint:errcheck
```

Although you could also close it and check the error after writing it to the file, but up to you 😉

Reply:
Done!